backend/api/routes/routes.go

```diff
@@ -107,6 +107,9 @@ func SetupRoutes(router *gin.Engine) {
 		// POST /survey/after_semester_survey/:user_id
 		surveyRoutes.POST("/:userId", application.PostTutorApplicationHandler)
 
+		// POST /survey/after_semester_survey/:user_id
+		surveyRoutes.POST("/:userId", application.UpdateTutorApplicationHandler)
+
```

Comment:
we cannot have two things with the same path and HTTP request type  we should change line 111 to PUT to reflect that we are updating not posting a new app